marketplace/deployer_util/validate_app_resource.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```

Comment:
This file was reverted in the master branch. Rm

Reply:
Merged from master again.